k8s/images/app/Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
 RUN apt-get update --fix-missing && apt-get -y install curl
 
-# Add the Cloud SDK distribution URI as a package source
-RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-xenial main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
```

Comment:
Did we use cloud SDK for something previously? just curious...

Reply:
We did for some live debugger functionality from GCP, but I never managed to get that working stably. I removed it for now since we don't need the SDK right now for anything.

Reply:
ah makes sense!